docs/example-react-hooks-useReducer.md

```diff
@@ -0,0 +1,119 @@
+Basic example showing how to test useReducer hook. 
+
+
+The component: changes stateprop1 from false to true which changes some text. 
```

Comment:
I'd suggest mentioning that the component changes the DOM, since this is the main benefit of using Testing Library tools - focusing on DOM changes and events, rather than component instances.

```suggestion
The component: changes some text depending on `stateprop1` value.
```